examples/ingestion-pipeline/requirements.txt

```diff
@@ -0,0 +1,7 @@
+apache-beam==2.24.0
+firebase-admin==4.4.0
+flickrapi==2.4.0
+#TODO: fix git+git://github.com... requirement
+
+-e git+https://github.com/googleinterns/ellipsis-step-internship.git@taltamir#egg=ingestion_pipeline_TALTAMIR&subdirectory=examples/ingestion-pipeline
+#https://github.com/googleinterns/ellipsis-step-internship/tree/taltamir/examples/ingestion-pipeline@taltamir#egg=ingestion_pipeline_TALTAMIR&subdirectory=ingestion-pipeline
```

Comment:
This is still not working, right? Can you try this: https://gist.github.com/inchoate/bd0ff7f609f57c85d9de8ff9d5586e30?

Reply:
It is working with the this-https://gist.github.com/inchoate/bd0ff7f609f57c85d9de8ff9d5586e30